terraform/modules/aws-brig-prekey-lock-event-queue-email-sending/resources.dynamodb.tf

```diff
@@ -0,0 +1,15 @@
+# FUTUREWORK: Potentially look at autoscaling for dynamoDB
```

Comment:
Right, I had a look at it: https://github.com/zinfra/backend-issues/issues/1319 but that was done with ansible. Once it makes its way to `prod`, I think we can adjust/adapt this.